R/common.R

```diff
@@ -189,10 +189,10 @@ registerFonts <- function() {
     # systemfonts::register_font(fontName, normalizePath(fontFile))
     # jaspGraphs::setGraphOption("family", fontName)
 
-    if (exists(".resultsFont"))
-      jaspGraphs::setGraphOption("family", .resultsFont)
+    if (exists(".resultFont"))
+      jaspGraphs::setGraphOption("family", .resultFont)
     else
-      warning("registerFonts was called but resultsFont does not exist!")
+      warning("registerFonts was called but resultFont does not exist!")
```

Comment:
Great that you updated the translation everywhere! However, this is exactly the kind of warning that needs `domain = NA` because it shouldn't be translated... That only makes our job harder when debugging and this warning isn't shown to users anyway.

Reply:
Don't worry,  we dont needs `domain = NA` anymore :-) now will also remove the extra translation from the po.

Reply:
I just went replace-all ^^